common/src/test/java/com/ericsson/bss/cassandra/ecaudit/common/chronicle/TestAuditRecordReadMarshallable.java

```diff
@@ -41,55 +43,50 @@
 @RunWith(MockitoJUnitRunner.StrictStubs.class)
 public class TestAuditRecordReadMarshallable
 {
-    @Mock
+    @Mock(lenient = true)
     private WireIn wireInMock;
 
+    private RecordValues defaultValues;
+    private AuditRecordReadMarshallable readMarshallable;
+
+    @Before
+    public void before() throws UnknownHostException
+    {
+        defaultValues = RecordValues.defaultValues();
+        readMarshallable = new AuditRecordReadMarshallable();
+    }
+
     @Test
     public void testReadSingleRecord() throws Exception
     {
-        givenNextRecordIs((short) 0, "ecaudit-single", 42L, InetAddress.getByName("1.2.3.4").getAddress(), InetAddress.getByName("5.6.7.8").getAddress(), "john", null, Status.ATTEMPT.toString(), "Some operation");
-
-        AuditRecordReadMarshallable readMarshallable = new AuditRecordReadMarshallable();
+        givenNextRecordIs(defaultValues);
 
         readMarshallable.readMarshallable(wireInMock);
 
         AuditRecord actualAuditRecord = readMarshallable.getAuditRecord();
 
-        assertThat(actualAuditRecord.getBatchId()).isEmpty();
-        assertThat(actualAuditRecord.getClientAddress()).isEqualTo(InetAddress.getByName("1.2.3.4"));
-        assertThat(actualAuditRecord.getCoordinatorAddress()).isEqualTo(InetAddress.getByName("5.6.7.8"));
-        assertThat(actualAuditRecord.getStatus()).isEqualTo(Status.ATTEMPT);
-        assertThat(actualAuditRecord.getOperation().getOperationString()).isEqualTo("Some operation");
-        assertThat(actualAuditRecord.getUser()).isEqualTo("john");
-        assertThat(actualAuditRecord.getTimestamp()).isEqualTo(42L);
+        assertThatRecordIsSame(actualAuditRecord, defaultValues);
     }
 
     @Test
     public void testReadBatchBatch() throws Exception
     {
-        givenNextRecordIs((short) 0, "ecaudit-batch", 42L, InetAddress.getByName("1.2.3.4").getAddress(), InetAddress.getByName("5.6.7.8").getAddress(), "john", UUID.fromString("bd92aeb1-3373-4d6a-b65a-0d60295f66c9"), Status.ATTEMPT.toString(), "Some operation");
-
-        AuditRecordReadMarshallable readMarshallable = new AuditRecordReadMarshallable();
+        RecordValues expectedValues = defaultValues
```

Comment:
Really liked this rewrite with the defaulValues.but...
Makes each test case short and readable, and it is clear how they differ from each other (not that much "noise"). 
Nice!

Reply:
Yeah, I thought so too. Updating a couple of other tests as well...